training/src/main/scala/com/airbnb/aerosolve/training/pipeline/GenericPipeline.scala

```diff
@@ -139,6 +139,23 @@ object GenericPipeline {
     val metrics = evalCompute(sc, config, cfgKey, isTrainingFunc)
 
     metrics.foreach(x => log.info(if (x._1 contains "THRESHOLD") x._1 else x.toString))
+
+    // Write metrics to result
+    try {
+      val result: String = confi.getConfig(cfgKey).getString("result")
```

Comment:
you will need the config of output result path, i.e. if it has output_result_path, then save result this way